inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/parser/result/FlinkSqlParseResult.java

```diff
@@ -58,7 +58,7 @@ public FlinkSqlParseResult(TableEnvironment tableEnv, List<String> createTableSq
     @Override
     public void execute() throws Exception {
         executeCreateTableSqls(createTableSqls);
-        executeLoadSqls(loadSqls).await();
+        executeLoadSqls(loadSqls);
```

Comment:
Why remove await()?

Reply:
Once in testcontainer. `jobManager.execInContainer("bash", "-c", "flink run -d -c org.apache.inlong.sort.Entrance /tmp/sort-dist.jar  --lightweight true --sql.script.file /tmp/kafka.sql");` will await commnd finish, but because awit, it will never compelete.
The reason for this design before is to facilitate testing，so once we want test in IDEA, add `await`, and we commit code we delete `await`.